src/content/graphos/enterprise/config.json

```
@@ -17,6 +17,7 @@
   ],
   "sidebar": {
     "Overview": "/",
+    "Reference Architecture": "/reference-architecture",
     "Self-Hosted Routing": {
       "Overview": "https://www.apollographql.com/docs/graphos/routing/self-hosted",
       "Setup": "https://www.apollographql.com/docs/graphos/quickstart/self-hosted",
```

After discussing this a little more with @shorgi , we felt the reference architecture page might be better suited under the Self-Hosted Routing section.

```suggestion
    "Self-Hosted Routing": {
      "Overview": "https://www.apollographql.com/docs/graphos/routing/self-hosted",
      "Setup": "https://www.apollographql.com/docs/graphos/quickstart/self-hosted",
      "Reference Architecture": "/reference-architecture",
```